src/app.tsx

```diff
@@ -29,6 +30,8 @@ i18n
       ru,
       "zh-TW": zhTW,
       "zh-CN": zhCN,
+      et,
+      "et-EE": et,
```

Comment:
Does Spotify give an option for both of these locales? I think if you just add it once, it should use it as a fallback if the corresponding one is in use?

Reply:
removed